plugins/Rockets/SnapshotTask.h

```diff
@@ -56,9 +56,8 @@ class SnapshotFunctor : public TaskFunctor
     SnapshotFunctor(Engine& engine, SnapshotParams&& params,
                     ImageGenerator& imageGenerator)
         : _params(std::move(params))
-        , _frameBuffer(engine.createFrameBuffer(_params.size,
-                                                FrameBufferFormat::rgba_i8,
-                                                true))
+        , _frameBuffer(engine.createFrameBuffer("snapshot", _params.size,
+                                                FrameBufferFormat::rgba_i8))
```

Comment:
Check with @favreau how he's generating stereo movies at the moment because it's not possible to get left/right eye images anymore (even less side by side).